### PR TITLE
Article: Skeleton

### PR DESCRIPTION
**Pull request by theflametrooper**
Refer #707

### CODE REVIEW

**Comment by alayek** [comment]
# LGTM :+1:

File: Skeleton.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/skeleton/2.0.4/skeleton.min.css">
+  </head>
+  <body>
+    <h1>Hello World</h1>
+    <!-- Add all HTML Code here -->
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+  </body>
+</html>
+```
+
```
Comment (abhisekp) [inline Skeleton.md]: Add a http://jsbin.com link with the template.

<kbd><a href="http://jsbin.com"> <strong>JSBin Demo ⮭</strong></a></kbd>

```md
<kbd><a href="http://jsbin.com"><img src="http://i.imgur.com/wBLOt8L.png"> <strong>JSBin Demo ⮭</strong></a></kbd>
```

**Comment by abhisekp** [comment]
@theflametrooper **File Rename:-**
```diff
- Skeleton.md
+ CSS-Framework-Skeleton.md
```

File: Skeleton.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+We have used a CDN in this example, but you can checkout other ways of installing Skeleton [here](https://github.com/dhg/Skeleton#getting-started).
+
+## Learning Resources
+
+- The official documentation for Skeleton is available [here](http://getskeleton.com/#grid).
```
Comment (abhisekp) [inline Skeleton.md]: - :point_right: [**`Official Documentation`**](http://getskeleton.com/#grid) of Skeleton

```md
- :point_right: [**`Official Documentation`**](http://getskeleton.com/#grid) of Skeleton
```

**Pull request merged**